README.md

```diff
@@ -1 +1,3 @@
 # Nuxeo Compound Documents
+
+This package is empty for now and will be used to structure compound documents packages for nuxeo.
```

Comment:
I would leave this empty or put TODO instead, to avoid forgetting about it later.
Or maybe put a simple description if we already know what a compound document is, see Epic and related Jira issues.

Reply:
Added some details from the master epic https://jira.nuxeo.com/browse/NXP-30767

Reply:
Cool, then I believe that this statement can be removed :)